tools/engine_tool/lib/src/commands/build_command.dart

```diff
@@ -0,0 +1,108 @@
+import 'package:engine_build_configs/engine_build_configs.dart';
+import 'package:path/path.dart' as path;
+import 'package:process_runner/process_runner.dart';
+
+import '../build_utils.dart';
+
+import 'command.dart';
+
+const String _configFlag = 'config';
+
+// Some wrinkles I'd like to iron out:
+//  - There can be multiple GlobalBuild objects with the same name. How
+//    should we deal with these conflicts? "mac_clang_tidy" and
+//    "mac_host_engine" both define "host_debug".
+//  - Naming: Should BuildConfig be BuilderConfig and GlobalBuild be BuildConfig?
+//  - The BuildConfig database seems to be lacking things I expected:
+//    - There is no "android_debug_*" GlobalBuild in any of the builders that run on mac?
```

Comment:
It could also be a sign that we have too many configurations (or too few on CI)

That being said, the easiest suggestion I think is to have local JSON files.

Reply:
> Another is to create some new json files that we only run locally and not on CI. Open to suggestions about what to do.

I lean strongly towards introducing new builders for local development. The CI builders run targets/tests that aren't necessary for local development, resulting in a slower inner loop. For example: producing engine .zip archives is rarely necessary when developing.

For that reason, Crank's builders are a fork of the CI builders: https://github.com/loic-sharma/crank/blob/4b114fed1668a383e1540b2bf4d29dc0b1dc1148/lib/builders.dart#L5-L359